### PR TITLE
feat: Add slowmode_delay attribute to discord.VoiceChannel

### PR DESCRIPTION
**Pull request by its-darsh**
## Summary

This PR added a new attribute for the `discord.VoiceChannel` which's `slowmode_delay` while it was missing, doc-strings has been updated.

## Information

- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting, examples, ...).

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [x] If code changes were made, then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why.
- [x] I have updated the changelog to include these changes.

### CODE REVIEW

File: discord/channel.py [hunk 1]
```diff
@@ -1482,6 +1484,12 @@ class VoiceChannel(discord.abc.Messageable, VocalGuildChannel):
     last_message_id: Optional[:class:`int`]
         The ID of the last message sent to this channel. It may not always point to an existing or valid message.
 
+    slowmode_delay: :class:`int`
+        The number of seconds a member must wait between sending messages
+        in this channel. A value of `0` denotes that it is disabled.
+        Bots and users with :attr:`~Permissions.manage_channels` or
+        :attr:`~Permissions.manage_messages` bypass slowmode.
+
         .. versionadded:: 2.0
```
Comment (plun1331) [inline discord/channel.py:1493]: This got shoved here, it should be under `last_message_id`

File: discord/channel.py [hunk 2]
```diff
@@ -1482,6 +1484,12 @@ class VoiceChannel(discord.abc.Messageable, VocalGuildChannel):
     last_message_id: Optional[:class:`int`]
         The ID of the last message sent to this channel. It may not always point to an existing or valid message.
 
+    slowmode_delay: :class:`int`
+        The number of seconds a member must wait between sending messages
+        in this channel. A value of `0` denotes that it is disabled.
+        Bots and users with :attr:`~Permissions.manage_channels` or
+        :attr:`~Permissions.manage_messages` bypass slowmode.
+
```
Comment (plun1331) [inline discord/channel.py:1492]: Add a versionadded for 2.5

File: discord/channel.py [hunk 3]
```diff
@@ -1791,6 +1799,7 @@ async def edit(
         overwrites: Mapping[Role | Member, PermissionOverwrite] = ...,
         rtc_region: VoiceRegion | None = ...,
         video_quality_mode: VideoQualityMode = ...,
+        slowmode_delay: int = ...,
```
Comment (plun1331) [inline discord/channel.py:1802]: Parameter needs to be documented

File: CHANGELOG.md [hunk 4]
```diff
@@ -62,6 +62,8 @@ These changes are available on the `master` branch, but have not yet been releas
   ([#2086](https://github.com/Pycord-Development/pycord/pull/2086))
 - Added new embedded activities, Gartic Phone and Jamspace.
   ([#2102](https://github.com/Pycord-Development/pycord/pull/2102))
+- Added `slowmode_delay` attribute to `discord.VoiceChannel`.
```
Comment (plun1331) [inline CHANGELOG.md:65]:
```suggestion
- Added `slowmode_delay` support to `VoiceChannel`.
```

**Review by plun1331 — DISMISSED** [review]

File: discord/channel.py (hunk 1 above)
Comment (DARSHTRON) [reply discord/channel.py:1493]: srry :3

**Review by JustaSqu1d — CHANGES_REQUESTED** [review]
Still missing version added?

**Comment by its-darsh** [comment]
it's been so long, if someone needed me please mention.

File: discord/channel.py (hunk 2 above)
Comment (JustaSqu1d) [reply discord/channel.py:1492]: > Add a versionadded for 2.5

**Review by JustaSqu1d — CHANGES_REQUESTED** [review]

File: discord/channel.py (hunk 2 above)
Comment (its-darsh) [reply discord/channel.py:1492]: forgot that, sorry

File: discord/channel.py [hunk 5]
```diff
@@ -1482,7 +1484,13 @@ class VoiceChannel(discord.abc.Messageable, VocalGuildChannel):
     last_message_id: Optional[:class:`int`]
         The ID of the last message sent to this channel. It may not always point to an existing or valid message.
 
-        .. versionadded:: 2.0
```
Comment (JustaSqu1d) [inline discord/channel.py:1485]: This should not be removed as `last_message_id` is for v2.0.

**Review by JustaSqu1d — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -75,6 +75,8 @@ These changes are available on the `master` branch, but have not yet been releas
   ([#2206](https://github.com/Pycord-Development/pycord/pull/2206))
 - Added function `Guild.delete_auto_moderation_rule`.
   ([#2153](https://github.com/Pycord-Development/pycord/pull/2153))
+- Added `slowmode_delay` support to `VoiceChannel`.
```
Comment (Dorukyum) [inline CHANGELOG.md:78]:
```suggestion
- Added `VoiceChannel.slowmode_delay`.
```

**Review by Lulalaby — APPROVED** [review]

**Review by Dorukyum — APPROVED** [review]

**Pull request merged**